src/main/resources/schemas/tekton.dev/v1alpha1_Pipeline.json

```diff
@@ -1258,8 +1253,8 @@
           "type": "array"
         },
         "timeout": {
-          "$schema": "http://json-schema.org/draft-04/schema#",
-          "$ref": "#/definitions/Duration"
+          "pattern": "^[-+]?([0-9]*(\\.[0-9]*)?(ns|us|µs|μs|ms|s|m|h))$",
```

Comment:
Not sure which is your idea on it but if we want to follow Go Duration doc it doesn't work correctly. For instance it fails with 1h3m